frontend/public/components/stateful-set.jsx

```diff
@@ -81,5 +82,5 @@ export const StatefulSetsPage = props => <ListPage {...props} ListComponent={Sta
 export const StatefulSetsDetailsPage = props => <DetailsPage
   {...props}
   menuActions={menuActions}
-  pages={[navFactory.details(Details), navFactory.editYaml(), navFactory.pods(), navFactory.envEditor(environmentComponent)]}
+  pages={[navFactory.details(Details), navFactory.editYaml(), navFactory.pods(), navFactory.envEditor(environmentComponent), navFactory.events(ResourceEventStream)]}
```

Comment:
Perhaps pull this one out `const pages = [...]` since there are so many items, consistent w/the above pages.